### ISSUE TITLE
Inline add attachment button is unlabeled in TalkBack 

### ISSUE THREAD

**Issue by ellenpoe** [signalapp/Signal-Android#8730]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The inline add attachment button that shows when the user is midway through composing a message and hasn't sent it yet is unlabeled. It should have a content description.

### Steps to reproduce
- Turn on TalkBack
- Compose a message and swipe right once
- Observe what's verbalized

**Actual result:** "Unlabled button" is verbalized
**Expected result:** "Add attachment" or "Add attachment, button" should be verbalized depending on system settings.

### Device info

**Device:** Pixel 2
**Android version:** Android 9
**Signal version:** built from HEAD

I have a fix for this ready.
